### PR TITLE
Gui preset Menu

### PR DESCRIPTION
**Pull request by tkolenbrander**
The preset menu is now fully scalable and functional (for as far as possible without camera communication).

### CODE REVIEW

**Comment by bvobart** [comment]
Even though the build passes, I get a compilation error, both from Eclipse and Maven test:
`[ERROR] /D:/Eclipse/Workspace/Context-TFP/src/main/java/nl/tudelft/contextproject/gui/PresetController.java:[106,23] cannot find symbol
  symbol:   method setOnAction((event)->{[...]; } })
  location: variable cameraSelecter of type javafx.scene.control.ChoiceBox<java.lang.Integer>`

**Comment by stevenmeijer** [comment]
@bvobart Not reproducible, both Eclipse and Maven run fine for me on this build

**Comment by tkolenbrander** [comment]
@bvobart Are you using the latest java version?

**Comment by bvobart** [comment]
@TH3steven @tkolenbrander The problem has been fixed by updating my Java version. It is apparently required to use Java 8 update 91 / 92.

File: src/main/java/nl/tudelft/contextproject/gui/PresetController.java [hunk 1]
```diff
@@ -1,34 +1,116 @@
 package main.java.nl.tudelft.contextproject.gui;
 
+import java.io.IOException;
```
Comment (bvobart) [inline src/main/java/nl/tudelft/contextproject/gui/PresetController.java]: CheckStyle gives errors about these imports, saying they are not in the wrong order.

**Comment by bvobart** [comment]
I think it should be possible to remove a preset from the list of presets in the Edit Presets window.

File: src/main/java/nl/tudelft/contextproject/gui/PresetController.java (hunk 1 above)
Comment (tkolenbrander) [inline src/main/java/nl/tudelft/contextproject/gui/PresetController.java]: I updated my checkstyle and the errors now show up in my eclipse. I will fix them right away!
EDIT [21:57]: The violations are now gone.

**Comment by tkolenbrander** [comment]
@bvobart I added the requested feature

**Comment by bvobart** [comment]
@tkolenbrander Thanks! I now approve this pull request.

**Comment by stevenmeijer** [comment]
Yup, looks good

**Comment by Ettatabe** [comment]
this pull request is as well approved by me since I don't get any errors and the code looks good. I will therefore merge this pull request and good job buddy.

**Pull request merged**